doc/radosgw/adminops.rst

```diff
@@ -10,6 +10,54 @@ mechanism. Some operations require that the user holds special administrative ca
 The response entity type (XML or JSON) may be specified as the 'format' option in the
 request and defaults to JSON if not specified.
 
+Info
+====
+
+Get RGW cluser/endpoint information.
+
+:caps: info=read
+
+
+Syntax
+~~~~~~
+
+::
+
+	GET /{admin}/info?format=json HTTP/1.1
+	Host: {fqdn}
+
+
+Request Parameters
+~~~~~~~~~~~~~~~~~~
+
+None.
+
+
+Response Entities
+~~~~~~~~~~~~~~~~~
+
+If successful, the response contains the user information.
+
+``info``
+
+:Description: A container for all returned information.
+:Type: Container
```

Comment:
i'm not 100% sure on this, so it needs testing, but..

a json document is either an object (i.e. a dictionary of key/values) or an array, and that doesn't include the name from `open_array_section("info")`. if you want to display this `"info"` part, i think you'd need to wrap it in another object section:
```c++
   formatter->open_object_section("dummy");                              // {
   formatter->open_object_section("info");                               //   "info" : {
   formatter->dump_string("cluster_id", store->get_cluster_id(this, y)); //     "cluster_id": "abcd"
   formatter->close_section();                                           //   }
   formatter->close_section();                                           // }
```
an example python client would read/decode this response like:
```python
import json

response = json.loads(input)
info = response['info']
fsid = info['cluster_id']
```
or we could choose to omit the "info" object and return only the `{ "cluster_id": "abcd" }` part

Reply:
I'd like to have the named section for explicitness, I'll try doing it this way.